tiledb/sm/c_api/tiledb.h

```diff
@@ -1112,6 +1112,9 @@ TILEDB_EXPORT void tiledb_config_free(tiledb_config_t** config);
  *    The S3 use of virtual addressing (`true` or `false`), if S3 is
  *    enabled. <br>
  *    **Default**: true
+ * - `vfs.s3.skip_init` <br>
+ *    Skip init for the S3 layer (`true` or `false`) <br>
```

Comment:
One last minor comment: let's explicitly name the `Aws::InitAPI` in the in-code documentation for this configuration parameter. If I were a developer who was finding a crash on `InitAPI`, a grep on `InitAPI` in my source code might lead me here.

Reply:
@joe-maley good idea!